src/test/data/JsonSerializableStoreMandoTest/invalidItemStoreMando.json

```diff
@@ -1,10 +1,10 @@
 {
   "items": [
     {
-      "name": "Hans Muster",
-      "quantity": "9482424",
-      "expirydate": "invalid@email!3e",
-      "location": "4th street"
+      "name": "Mustard",
+      "quantity": "9",
+      "expirydate": "20010-1-1",
+      "location": "Refrigerator"
```

Comment:
One suggestion I have for these JSON files is that maybe we should rearrange the sequence to match our display sequence to make things more neat and tidy :) so instead of the current name, quantity, exdate, location, we put name followed by location and so on based on our current display.

Reply:
Personally, I think this is okay since this is within the test folder and we are abstracting the value through key-value mapping. Furthermore, we do not many variables so as long as each JSON object follows the same sequence I am good.

Reply:
Agreed with Wei Hao. I think it is fine to leave it as it is since there is no fixed input format and prefixes can be specified in any order.